docs/pages/versions/unversioned/sdk/camera.md

```diff
@@ -261,6 +261,7 @@ Starts recording a video that will be saved to cache directory. Videos are rotat
   - **maxDuration (_number_)** -- Maximum video duration in seconds.
   - **maxFileSize (_number_)** -- Maximum video file size in bytes.
   - **mute (_boolean_)** -- If present, video will be recorded with no sound.
+  - **setVideoMirrored (_boolean_)** -- (iOS only; on Android, this is handled in the user's device settings) If `true`, the recorded video will be flipped along the vertical axis. iOS flips videos recorded with the front camera by default, but you can reverse that back by setting this to `true`.
```

Comment:
I don't like this name at all ⛔️. `setVideoMirrored` suggests it is as setter function that can be called subsequently (correctly named function/method should contain verb and `set` is more or less reserved for parameter setter that might be called on some object) while we need just a parameter here.

`videoMirrored` is sufficient and a better name. Having `videoMirrored` plays nicely with underlying Apple's API, because you get `[connection setVideoMirrored:videoMirrored]`. One can argue that `isVideoMirrored` might be even better name (matter of explicit verbosity though). 

`videoMirrored` might not be a perfect one though, because we have `mute` parameter instead of `muted` and now I'm a little confused 😕😞

When I think about nicely created `Camera.recordAsync()` API then I image something like this: let's call `Camera.recordAsync` with `quality` set to `QUALITY.STH`, `maxDuration` set to `NUMBER_OF_SECONDS` and `maxFileSize` set to `NUMBER_OF_BYTES`. Additionally I want my `video`  to be `muted` and possibly `mirrored`.

And because the `API from my dream` isn't achievable here (because of `mute` parameter that conflicts with my dreamed `muted` parameter name) I wonder whether `mirrorVideo` wouldn't be even a better name for this parameter to go along with `mute` parameter name. cc @esamelson

Reply:
Thanks for the explanation, hadn't thought of the fact that the current name makes it look like a setter. 

I agree that `mirror` or `mirrored` are both better. I guess the question is- would we rather have consistency (i.e. `mute` and `mirror`), **or** sacrifice consistent param names (i.e. use `mute` and `mirrored`) to make small steps towards making this match the API of your dreams. Personally I would say we stick to the former (`mute` and `mirror`) unless we want to deprecated `mute` in favor `muted` and add logic to handle the old way for an SDK or two.

Reply:
Let's be consistent for now 👍